debian/changelog

```diff
@@ -1,3 +1,13 @@
+eos-chrome-plugin-updater (1.0.2-1) eos; urgency=low
+
+  * New upstream release 1.0.2
+  * Install widevinecdm manifest.
```

Comment:
```suggestion
  * Install symlink for widevinecdm manifest.
```

Reply:
Well, being really pedantic it would be "Install manifest and create symlink for chromium" hehe :).